.github/workflows/ci.yaml

```diff
@@ -71,3 +71,15 @@ jobs:
           provider: lxd
       - name: Run integration database tests
         run: tox -e integration-database
+  integration-test-lxd-mysql:
+    name: Integration tests for mysql relation (lxd)
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup operator environment
+        uses: charmed-kubernetes/actions-operator@main
+        with:
+          provider: lxd
+      - name: Run integration database tests
+        run: tox -e integration-mariadb
```

Comment:
recommend renaming `integration-mariadb` -> `integration-myql-interface` to avoid any confusion